tests/integration/test_workflow_yaml_validation.py

```
@@ -0,0 +1,612 @@
+            with open(workflow_file, "r") as f:
+                data = yaml.safe_load(f)
+
+            name = data.get("name", "")
+            assert name and len(name) > 3, (
+                f"{workflow_file.name} has empty or too short name: '{name}'"
+            )
+            assert not name.isupper(), (
+                f"{workflow_file.name} name should not be all uppercase: '{name}'"
+            )
+
+    def test_workflow_names_are_descriptive(self, workflow_files: List[Path]):
```

Ruff (0.15.1)

[error] Redefinition of unused `test_workflow_names_are_descriptive` from line 70: `test_workflow_names_are_descriptive` redefined here

(F811)